### DISCUSSION TITLE
Nova 4 Slug field Bug

### DISCUSSION THREAD

**Discussion by stepanenko3** [spatie/nova-translatable#76] [Ideas]
Not working with Nova 4 Slug field
And I think with dependsOn doesn't working either

**Comment by freekmurze** [comment]
I'm not using it in combination with a slug field myself.

I'm open to PR that fixes this.

**Comment by chimit** [comment]
Have the same problem. I believe it's quite a common case:

```php
Translatable::make([
    Text::make('Name'),
]),

Slug::make('Slug')->from('Name'),
```

@freekmurze, please, add support for it 🙏

**Reply by samigitz** [reply]
if anyone having this issue, try below

`Slug::make('Slug')
                ->from('translations_title_en')`

This is working
